### ISSUE TITLE
Addon Message

### ISSUE THREAD

**Issue by mgfarm** [DarkTrick/AnkiTagSelector#2]
I am helpless with computers and got the following message when I went to open Anki:
An add-on you installed failed to load. If problems persist, please go to the Tools>Add-ons menu, and disable or delete the add-on.

When loading '⁨TagSelectorV2⁩':
⁨Traceback (most recent call last):
  File "/Users/maryfarmer/Library/Application Support/Anki2/addons21/1022577188/__init__.py", line 806, in main_wrapFunctions
    AddCards._reject = wrap(AddCards._reject, main_onMaybeCloseAddCardsDialog,"before")
AttributeError: type object 'AddCards' has no attribute '_reject'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "aqt.addons", line 244, in loadAddons
  File "/Users/maryfarmer/Library/Application Support/Anki2/addons21/1022577188/__init__.py", line 842, in <module>
    main_wrapFunctions()
  File "/Users/maryfarmer/Library/Application Support/Anki2/addons21/1022577188/__init__.py", line 811, in main_wrapFunctions
    AddCards.reject = wrap(AddCards.reject, main_onMaybeCloseAddCardsDialog,"before")
AttributeError: type object 'AddCards' has no attribute 'reject'

**Comment by DarkTrick** [comment]
Operating system? Operating system version? Anki version?

**Comment by mgfarm** [comment]
MacOS 12.2, Anki (apple silicon 2.1.54 qt6) or (intel 2.1.54 qt5) or (intel 2.1.54 qt6) - I tried all three

**Comment by DarkTrick** [comment]
Thank you for the feedback. I will take a look at this and hope(?) I can reproduce it on my computer.
However, this will probably take a couple of weeks.

**Comment by tobiasgreiser** [comment]
I get the exact same error message on startup.

 
```
    Anki 2.1.54 (b6a7760c) Python 3.9.10 Qt 5.15.2 PyQt 5.15.5
    Platform: Linux (Ubuntu 20.04.5 LTS)
    Flags: frz=True ao=True sv=3
    Add-ons, last update check: 2022-09-14 11:41:55
    
    ===Add-ons (active)===
    (add-on provided name [Add-on folder, installed at, version, is config changed])
    TagSelectorV2 ['1022577188', 2021-08-07T13:28, 'None', '']
    
    ===IDs of active AnkiWeb add-ons===
    1022577188
    
    ===Add-ons (inactive)===
    (add-on provided name [Add-on folder, installed at, version, is config changed])
    Anki Simulator ['817108664', 2021-08-07T21:24, 'None', '']
    Image Occlusion Enhanced for Anki 21 alpha ['1374772155', 2022-04-09T09:15, 'None', '']
    Review Heatmap ['1771074083', 2022-06-30T03:43, 'None', '']
```

`

**Comment by DarkTrick** [comment]
Fixed with [version 24.6.3.1](https://github.com/DarkTrick/AnkiTagSelector/releases/tag/v24.6.3.1).
